### ISSUE TITLE
Prepare validation set on HPC

### ISSUE THREAD

**Issue by kutayeroglu** [kutayeroglu/ijepa#37]
> The error's cause is the flat structure of the validation set.
> It was most likely downloaded from Kaggle, see the following link for people having a similar problem: https://discuss.pytorch.org/t/issues-with-dataloader-for-imagenet-should-i-use-datasets-imagefolder-or-datasets-imagenet/115742
> 
> I need to re-organize the validation data.
> Instead of re-organizing the shared dataset, I will download the validation data and reorganize it in my personal storage area.
> 
> 
>  

 _Originally posted by @kutayeroglu in [#32](https://github.com/kutayeroglu/ijepa/issues/32#issuecomment-3517795780)_

I plan to utilize https://gist.github.com/antoinebrl/7d00d5cb6c95ef194c737392ef7e476a for this. 

Will download validation set with `wget https://image-net.org/data/ILSVRC/2012/ILSVRC2012_img_val.tar --no-check-certificate`
